docs/index.md

```diff
@@ -0,0 +1,14 @@
+# Welcome to era5cli’s documentation!
```

Comment:
Would it be possible to get rid of the `about` header in the left side bar, as it is the same as `home`?